core/controllers/feedback_test.py

```diff
@@ -220,15 +220,15 @@ def test_post_message_to_existing_thread(self):
             set(response_dict['messages'][0].keys()),
             set(EXPECTED_MESSAGE_KEYS))
         self.assertDictContainsSubset({
-            'author_username': self.EDITOR_USERNAME,
+            'author_id': self.editor_id,
```

Comment:
Why do we return ID here and not username?

Reply:
previously I missed it but updated this time.